test/RibbonTreasuryVault.ts

```diff
@@ -3253,6 +3253,11 @@ function behavesLikeRibbonOptionsVault(params: {
           .to.emit(vault, "OpenShort")
           .withArgs(secondOptionAddress, depositAmount.mul(2), keeper);
 
+        firstOptionPremium =
```

Comment:
think current Error: VM Exception while processing transaction: reverted with reason string 'limit price not better than mimimal offer' error is because block is on friday (not thursday).

this is temporary fix